packages/cli-core/src/format/format.ts

```diff
@@ -30,7 +30,18 @@ export class Format {
     }
     return cyan(`${this.packageManagerRunnerName} amplify ${command}`);
   };
-  error = (message: string) => red(message);
+  error = (error: string | Error | unknown) => {
+    if (error instanceof Error) {
+      return red(`${error.name}: ${error.message}`);
+    } else if (typeof error === 'string') {
+      return red(error);
+    }
+    try {
+      return red(JSON.stringify(error, null, 2));
+    } catch {
+      return red('Invalid Error');
```

Comment:
Should we change the wording here or keep the original wording: "Unknown Error"? https://github.com/aws-amplify/amplify-backend/blob/1f38466da73800d9f57cbd1fd35fd98bf95227d5/packages/cli/src/commands/sandbox/sandbox_event_handler_factory.ts#L58

Reply:
`invalid` is confusing for customers as it might mean they did something that was invalid but in reality we have no idea what went wrong.

In this case however, you should capture the `error` from the try catch and print that as well along with `unknown error`